changelogs/OUTCOME_ATH_DICTIONARY_CHANGELOG.md

```diff
@@ -60,7 +78,101 @@ We define here in what versions of the data dictionaries contains which tables
+| crve_ | decimal | Repeated measures of child‘s central retinal venular equivalent (CRVE) (mean value from right and left fundus): rmccrve_: assessed within one year of childbirth (>0 year and <1 year) rmccrve_1: assessed when the child was aged between ≥ 1 year and < 2 years … rmccrve_17: assessed when the child was aged between ≥ 17 years and < 18 years |
+| avr_ | decimal | Repeated measures of child‘s arteriole-to-venule ratio (AVR) (mean value from right and left fundus): rmcavr_ assessed within one year of childbirth (>0 year and <1 year) rmcavr_1: assessed when the child was aged between ≥ 1 year and < 2 years … rmcavr_17: assessed when the child was aged between ≥ 17 years and < 18 years |
+| tortuosity_ | decimal | Repeated measures of child‘s tortuosity index (mean value from right and left fundus): rmcavr_ assessed within one year of childbirth (>0 year and <1 year) rmcavr_1: assessed when the child was aged between ≥ 1 year and < 2 years … rmcavr_17: assessed when the child was aged between ≥ 17 years and < 18 years |
+| rmc_age_ | decimal | Exact age of the child (in years) when arteriole-to-venule ratio (AVR) vas measured for rmcavr_, rmcavr_1, etc. |
+| rmcmes_ | integer | Method used to measure retinal microcirculation parameters (i.e. CRAE, CRVE, and AVR) for each measurement (e.g. rmcrve_, rmcrve_1, rmcrve_2 etc.) |
+| mets_ idefics_ | integer | Repeated classification of pediatric MetS based on the IDEFICS definition (pediatric MetS at a monitoring level when the values of at least three of the components of MetS exceed the 90th percentile) |
+| mets_score_idefics_ | decimal | Continuous pediatric MetS score combining all components based on the IDEFICS z-scores |
+| cvd_risk_score_ | decimal | Continuous pediatric CVD score combining three MetS components (not including the HOMA-IR/glucose component) based on the IDEFICS z-scores |
+| wc_idefics_perc_ | decimal | Repeated WC percentiles based on the IDEFICS study reference values |
+| sbp_idefics_perc_ | decimal | Repeated SBP percentiles based on the IDEFICS study reference values |
+| dbp_idefics_perc_ | decimal | Repeated DBP percentiles based on the IDEFICS study reference values |
+| triglyceride_idefics_perc_ | decimal | Repeated triglycerides percentiles based on the IDEFICS study reference values |
+| hdl_idefics_perc_ | decimal | Repeated HDL-C percentiles based on the IDEFICS study reference values |
+| homa_ir_ | decimal | Repeated HOMA-IR values |
+| homa_ir_idefics_perc_ | decimal | Repeated HOMA-IR percentiles based on the IDEFICS study reference values |
+| glucose_idefics_perc_ | decimal | Repeated glucose percentiles based on the IDEFICS study reference values |
+| mets_age_ | decimal | Exact age of the child (in years) when the MetS components were measured. |
+| alt_ | decimal | Repeated measures of child’s |
```

Comment:
I think the labels for these four variables (alt_, ast_, ggt_ and ck18_) are incomplete? 
'_Repeated measures of the child's.._.'?

Reply:
The yearly variables `alt_`  `ast_` `ggt_` `ck18_`  seems to be missing part of their labels @jmu82 could you add these to outcome_ath/1_0/1_0_yearly_rep.xlsx?

Reply:
Yes, this should be: alt_	decimal	U/L	Repeated measures of child’s alanine aminotrasferase concentrations
ast_	decimal	U/L	Repeated measures of child’s aspartate aminotrasferase concentrations
ggt_	decimal	U/L	Repeated measures of child’s gamma-glutamyl transferase concentrations
ck18_	decimal	U/L	Repeated measures of child’s Cytokeratin-18 concentrations